src/App.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import './App.css';
+import data from './data/resume.json';
+
+function BuildBasicsElement(basics) {
+  /*
+    todo: Elements in basic info remaining below.
+    For profiles, would have considered creating a for loop
+    to get through each profile.
+    "profiles": [{
+      "network": "Twitter",
+      "username": "john",
+      "url": "http://twitter.com/john"
+    }]
+  */
+  const { location } = basics;
+  return (
+    <div className="App" align="left">
+      <h1>{basics.name}</h1>
+      <h3>{basics.label}</h3>
```

Comment:
It's best to use H1, H2 etc for the semantic use - not for styling / the default size that comes with the element. This would probably work well as an H2, and so on and so forth.